runtime/help/colors.md

```diff
@@ -26,23 +26,23 @@ Modern terminals tend to have a palette of 16 user-configurable colors (these
 colors can often be configured in the terminal preferences), and additional
 color support comes in three flavors.
 
-* 16-color: A colorscheme that uses the 16 default colors will always work but
+* `16-color`: A colorscheme that uses the 16 default colors will always work but
```

Comment:
For 16-color, 256-color and true-color the backquotes might be a bit misleading, since these are not "proper names" (option names, terminal names etc)